app/Components/Sudoku Board/SudokuBoard.tsx

```diff
@@ -474,44 +587,47 @@ const SudokuBoard = (props: SudokuBoardProps) => {
       return;
     }
 
-    let newCol = sudokuBoard.selectedCell.c;
-    let newRow = sudokuBoard.selectedCell.r;
-    switch (inputValue) {
-      case "Delete":
-      case "Backspace":
-      case "0":
-      case "e":
-      case "E":
-        eraseSelected();
-        return;
-      // below cases do not return to allow for update of selected cell
-      // todo create function for updating selectedCell for below cases to call
-      case "ArrowLeft":
-      case "a":
-      case "A":
-        newCol = wrapDigit(newCol - 1);
-        break;
-      case "ArrowRight":
-      case "d":
-      case "D":
-        newCol = wrapDigit(newCol + 1);
-        break;
-      case "ArrowUp":
-      case "w":
-      case "W":
-        newRow = wrapDigit(newRow - 1);
-        break;
-      case "ArrowDown":
-      case "s":
-      case "S":
-        newRow = wrapDigit(newRow + 1);
-        break;
-      default:
-        return;
+    for (let i = 0; i < sudokuBoard.selectedCell.length; i++) {
```

Comment:
possibly change to selectedCells instead of selectedCell to make it clearer what length is and what iterating over it is doing

Reply:
Good idea, I renamed to selectedCells